apps/member-profile/app/shared/components/company-review.tsx

```diff
@@ -263,3 +275,32 @@ function CompanyReviewText({ text }: Pick<CompanyReviewProps, 'text'>) {
 CompanyReview.List = function List({ children }: PropsWithChildren) {
   return <ul className="flex flex-col gap-4">{children}</ul>;
 };
+
+function UpvoteCompanyReviewButton({
+  id,
+  upvoted,
+  upvotes,
+}: Pick<CompanyReviewProps, 'id' | 'upvoted' | 'upvotes'>) {
+  const fetcher = useFetcher();
+
+  const action = upvoted
+    ? `/api/company-review/${id}/downvote`
```

Comment:
Don't need downvote reference here-whenever button is pressed it will count as an upvote